code/modules/medical/genetics/bioEffects/beneficial.dm

```diff
@@ -310,6 +310,29 @@
 	icon_state  = "dead"
 	isBad = 1
 
+/datum/bioEffect/noir
+	name = "Noir"
+	desc = "The subject generates a light-defying aura, equalizing photons in such a way that make them look completely grayscale."
+	id = "noir"
+	effectType = EFFECT_TYPE_POWER
```

Comment:
I'm pretty sure POWER is reserved for activated abilities.

Reply:
Ok, removed that line from both  Achromia and Noir.